frontend/src/components/HomeGuide/index.tsx

```diff
@@ -39,39 +37,19 @@ const GuideButton = styled(Button)<ButtonProps>({
 export const HomeGuide = () => {
   const navigate = useNavigate()
   const dispatch = useAppDispatch()
-  const [validConfig, setValidConfig] = useState(true)
-
   const getImportFileElement = () => document.getElementById('importJson') as HTMLInputElement
 
-  const isValidImportedConfig = (configStr: string) => {
-    try {
-      const importedConfig = JSON.parse(configStr)
-      const {
-        projectName,
-        metrics,
-        dateRange: { startDate, endDate },
-      } = importedConfig
-      return !!projectName && !!startDate && !!endDate && metrics.length > 0
-    } catch {
-      return false
-    }
-  }
-
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const input = e.target.files?.[0]
     const reader = new FileReader()
     if (input) {
       reader.onload = () => {
         if (reader.result && typeof reader.result === 'string') {
-          if (isValidImportedConfig(reader.result)) {
-            dispatch(updateProjectCreatedState(false))
-            const importedConfig = JSON.parse(reader.result)
-            dispatch(updateBasicConfigState(importedConfig))
-            dispatch(updateMetricsImportedData(importedConfig))
-            navigate('/metrics')
-          } else {
-            setValidConfig(false)
-          }
+          dispatch(updateProjectCreatedState(false))
```

Comment:
How to handle the invalid JSON file?
PLZ see my code and change this logic

Reply:
done